### DISCUSSION TITLE
problem with INSTA devices

### DISCUSSION THREAD

**Discussion by emichel11** [Koenkk/zigbee2mqtt#8674] [Devices]
Hi,
I have 2 device from Insta - modell 57008000 and 57005000. The first one is a window cover, the second is a switch.
The modell 5700800 is supported, however 5700500 is not.
So at this stage both device are handled as 5700800

I have now added the device in data: 
const definition = {
	    zigbeeModel: ['Generic UP Device'], 
	    model: '57005000', 
	    vendor: 'Insta', 
	    description: 'Switch',
	    extend: extend.switch(),
	    configure: async (device, coordinatorEndpoint, logger) => {
	          const endpoint = device.getEndpoint(1);
	          await reporting.bind(endpoint, coordinatorEndpoint, ['genOnOff']);
	          await reporting.onOff(endpoint);
            },
};

module.exports = definition;

Now both controller are identified as 5700500 - the switch works, the window cover not.
Can somehelp me how I can differ these devices?

**Comment by K-r-i-s-t-i-a-n** [comment]
Hi,

same here, I have a model 57008000 and 57004000. Added an external converter but both devices use the same converter for 57008000. 

Is it correct, that Z2M does not use the converters "model: xxx" line to identify the devices? Are there other properties of the device that can be used to identify or do we need to wait until Insta changes the zigbeeModel (see https://github.com/Koenkk/zigbee-herdsman-converters/pull/2474)?

Thanks
Kris
